atlas/src/join_table.rs

```diff
@@ -0,0 +1,324 @@
+use sqlx::{pool::PoolConnection, Postgres};
+// JOin roads and featureclass, returns road_id, fid and fclass
```

Comment:
```suggestion
/// Join roads and featureclass, returns road_id, fid and fclass
```
use triple slash for doc comments